ouroboros-consensus/test/consensus-test/Test/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -611,6 +691,9 @@ instance Arbitrary ChainSyncClientSetup where
             , lastTick (getServerUpdates serverUpdates)  - 1
             ]
     , startTick' <- [1..min startTick maxStartTick]
+    ] <>
+    [ cscs { clientSlowBy = clockSkewInSeconds 0 }
+    | clientSlowBy /= clockSkewInSeconds 0
     ]
 
 instance Show ChainSyncClientSetup where
```

Comment:
Done, thanks. I also changed it to a proper `Show` instance (ie copy-pasteable) since that's useful for QuickCheck counterexamples. I relegated this `condense` stuff to a separate call to `counterexample`.